src/averageAges.js

```diff
@@ -15,11 +19,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const malePeople = people.filter(person =>
+    person.sex === MALE
+    && (century === undefined
+    || century === Math.ceil(person.died / CENTURY)));
```

Comment:
```suggestion
  const malePeople = people.filter(({ sex, died })=> (
    sex === MALE && (!century || century === Math.ceil(died / CENTURY))
  ));
```

Reply:
But would be even better:
<img width="1054" alt="image" src="https://github.com/mate-academy/js_average-ages/assets/86789161/b69bc21d-ca5b-44f7-97e5-cc96dab76450">